presto-main/src/main/java/io/prestosql/cost/StatsUtil.java

```diff
@@ -40,8 +40,8 @@ static OptionalDouble toStatsRepresentation(Metadata metadata, Session session,
     {
         if (convertibleToDoubleWithCast(type)) {
             InterpretedFunctionInvoker functionInvoker = new InterpretedFunctionInvoker(metadata);
-            Signature castSignature = metadata.getCoercion(type, DoubleType.DOUBLE);
-            return OptionalDouble.of((double) functionInvoker.invoke(castSignature, session.toConnectorSession(), singletonList(value)));
+            ResolvedFunction castFunction = metadata.getCoercion(type, DoubleType.DOUBLE);
```

Comment:
static import

Reply:
I added a commit at the end to clean up this class.